apps/ui/src/components/Encrypted.vue

```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts">
+withDefaults(
+  defineProps<{
+    encrypted: boolean;
+    iconOnly?: boolean;
+  }>(),
+  { encrypted: false, iconOnly: false }
+);
+</script>
+
+<template>
+  <div
+    v-if="encrypted"
+    v-tippy="{
```

Comment:
We should use `v-tippy` inside `UiTooltip` and use `UiTooltip` here instead I think

Reply:
Will move everything back as UiTooltip, and do the v-tippy migration in a separate PR